runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -65,11 +65,33 @@ default String filteredRegex()
 		return "";
 	}
 
+	@ConfigItem(
+		keyName = "filteredNames",
+		name = "Filtered Names",
+		description = "List of filtered names, seperated by commas",
+		position = 4
+	)
+	default String filteredNames()
+	{
+		return "";
+	}
+
+	@ConfigItem(
+		keyName = "filteredNamesRegex",
+		name = "Filtered Names Regex",
+		description = "List of filtered regular expressions for usernames, one per line",
+		position = 5
+	)
+	default String filteredNamesRegex()
```

Comment:
Any reason these can't be the same config?

Reply:
i dont think so, will change